support/qa/phpstan.neon

```diff
@@ -5,3 +5,14 @@ parameters:
     level: 7
     paths:
         - ../../src
+    ignoreErrors:
+        -
+            message: '#Method [a-zA-Z0-9\\_]+::[a-zA-Z0-9]+\(\) has parameter \$context#'
+            path: '**/Infrastructure/Serializer/SessionNormalizer.php'
+        -
+            message: '#Method [a-zA-Z0-9\\_]+::[a-zA-Z0-9]+\(\) return type has no value type specified in iterable type array#'
+            path: '**/Infrastructure/Serializer/SessionNormalizer.php'
+        -
+            message: '#Method [a-zA-Z0-9\\_]+::[a-zA-Z0-9]+\(\) return type with generic class ArrayObject does not specify its types: TKey, TValue#'
+            path: '**/Infrastructure/Serializer/SessionNormalizer.php'
```

Comment:
Wouldn't these exceptions be easily removed if you'd write the docblock (currently inherited from parent) yourself?

Reply:
AFAIK, the phpdoc can be extended, not overwrited. I waste a lot of time trying to avoid this issue add a local `@param`. But inherit docblock is wrong and I haven't found a way to avoid it.

I can not remove `inheritdoc` tag, because then phpcs and phpmd fails too. Is this or add annotations to avoid phpcs and phpmd errors.

Maybe using psalm instead phpstan...

Reply:
If its up to me, just leave the exceptions in place. Let's stay alert if this comes back more often. If it does we can always switch to Psalm, or configure Stan in a less strict manner.